motion/p5js/cones/index.js

```diff
@@ -4,7 +4,7 @@ let y = 0;
 let increment = 1;
 
 function setup() {
-  createCanvas(300, 300, WEBGL);
+  createCanvas(500, 500, WEBGL);
```

Comment:
The canvas needs to be assigned to div. Please change the code to this:  

```
  var canvas = createCanvas(500, 500, WEBGL);
  canvas.parent('sketch');
```

This allows to display it directly in the site.